packages/early-boot-config/Cargo.toml

```diff
@@ -10,6 +10,8 @@ path = "../packages.rs"
 
 [package.metadata.build-package]
 source-groups = [
+    "api",
+    "imdsclient",
```

Comment:
Let's just list `early-boot-config` here instead of the individual subdirectories.

Appears to be missing `constants`, `generate-readme`, `imdsclient`,  ...

The brute force way to generate this list would be:
* copy the sources tree to another location
* delete everything from the sources tree that isn't a directory in `source-groups`
* trim the workspace `Cargo.toml` and regenerate `Cargo.lock`
* see if the package still builds

`static-pods` and `netdog` probably also need to be revisited with an eye to completing the dependency picture.